.github/workflows/ci.yaml

```diff
@@ -0,0 +1,53 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - '[0-9]+.[0-9]+'
+  pull_request:
+
+jobs:
+  tests:
+    name: Tests
+    runs-on: "ubuntu-20.04"
+    timeout-minutes: 10
+    continue-on-error: ${{ matrix.experimental }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php:
+          - '7.3'
```

Comment:
Not sure if it's still needed

Reply:
Not if we keep the `main` branch as target - but it could be introduced to the 1.0 branch (relevant for DXP 3.3) where it would be useful.

Reply:
`composer.json` still has `^7.3`.

Reply:
Wrong target branch btw, good catch.

A CI that catches issues like this would be really good, as it happens often.